CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.25.0
 
 * Allow a custom password to be set for SCRAM-SHA-512 users by referencing a secret in the `KafkaUser` resource
+* Annotate Cluster Operator resource metrics by a namespace label
```

Comment:
Should it be in the changes section below? That way people looking for compatibility changes will be more likely to notice.